src/main/resources/db/changelog/changes/v1.1.0/sql/create-mat-view-drv-inventory-statistical-code-full.sql

```diff
@@ -0,0 +1,11 @@
+DROP MATERIALIZED VIEW IF EXISTS drv_inventory_statistical_code_full;
+
+CREATE MATERIALIZED VIEW drv_inventory_statistical_code_full AS
+SELECT
+	statcode.id AS "id",
+	CONCAT(stattype.jsonb ->> 'name', ': ', statcode.jsonb ->> 'code', ' - ', statcode.jsonb ->> 'name') as "statistical_code"
```

Comment:
Nitiest of nits to pick:

```suggestion
	CONCAT(stattype.jsonb ->> 'name', ': ', statcode.jsonb ->> 'code', ' - ', statcode.jsonb ->> 'name') AS "statistical_code"
```
(just to match the formatting in the rest of the file, where keywords are all caps)